modin/pandas/test/test_series.py

```diff
@@ -3570,8 +3424,12 @@ def test_tolist(data):
 
 
 @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-@pytest.mark.parametrize("func", agg_func_values, ids=agg_func_keys)
-def test_transform(data, func):
+@pytest.mark.parametrize(
+    "func", [lambda x: x + 1, [np.sqrt, np.exp]], ids=["lambda", "list_udfs"]
+)
+def test_transform(data, func, request):
+    if "list_udfs" in request.node.callspec.id:
+        pytest.xfail(reason="FIXME: Modin failed")
```

Comment:
Please create an issue for this.

Reply:
done

Reply:
https://github.com/modin-project/modin/issues/6998